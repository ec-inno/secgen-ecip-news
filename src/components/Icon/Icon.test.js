import React from 'react';
import renderer from 'react-test-renderer';
import Icon from './Icon';

describe('Icon', () => {
  it('renders correctly', () => {
    const props = { shape: 'ui-download' };
    const tree = renderer.create(<Icon {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
