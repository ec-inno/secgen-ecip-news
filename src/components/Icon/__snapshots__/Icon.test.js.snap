// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders correctly 1`] = `
<svg
  aria-hidden="true"
  className="ecl-icon ecl-icon--m"
  focusable="false"
>
  
  
  <use
    xlinkHref="file#ui-download"
  />
</svg>
`;
